### DISCUSSION TITLE
Lead screw equivalents?

### DISCUSSION THREAD

**Discussion by ClearAirTurbulence3D** [bassamanator/everything-sovol-sv06#15] [General]
I looked for replacement lead screws on Amazon, McMaster-Carr and OpenBuild and most lead screws have 4 starts, including the more common ones. I haven't checked on Aliexpress.

**Comment by bassamanator** [comment]
You need to look on Aliexpress. If you're replacing lead screws, I highly recommend you get 2mm lead lead screws (the repetition is intentional).

375mm is also not a standard size, so you may have to request this from the seller, many will be happy to oblige.
